smoderp2d/runners/arcgis.py

```diff
@@ -0,0 +1,13 @@
+import os
+from smoderp2d.runners.base import Runner
+
+class ArcGisRunner(Runner):
+    """TODO."""
+
+    def __init__(self):
+        os.environ['ESRIACTIVEINSTALLATION'] = '1'
+        super(ArcGisRunner, self).__init__()
+
+    def _get_provider(self):
```

Comment:
@landam: Don't you want to add a docstring here?

Reply:
Solved by 9accb4c8